### ISSUE TITLE
VS Code breaks intellisense when working with multiple-projects Angular workspace

### ISSUE THREAD

**Issue by graphicsxp** [microsoft/vscode#112451]
- VSCode Version: 1.52.0
- OS Version: Windows 10

Steps to Reproduce:

1. create a multi-projects Angular workspace
2. for each project, create a path in the tsconfig.app.json file that points to the app folder

Consider the following multi-projects Angular workspace 
![image](https://user-images.githubusercontent.com/4160415/102064289-68d5f180-3df7-11eb-9f7b-5636a6119a92.png)
All projects share the tsconfig.json file. 

Each project has its own tsconfig file that extends the root one. For instance :  
![image](https://user-images.githubusercontent.com/4160415/102065720-34fbcb80-3df9-11eb-95a3-5d02c565557a.png)

In the  ´tsconfig.app.json´  file I have created a path to the project ´app´ folder :  
![image](https://user-images.githubusercontent.com/4160415/102064544-ba7e7c00-3df7-11eb-8520-f2f7c3ffcdfe.png)
This can be called in any of the ts file of my projects :   

![image](https://user-images.githubusercontent.com/4160415/102064601-cc601f00-3df7-11eb-8439-a8044f63f766.png)

But VS Code does not like it and I get the error : `Cannot find module '@author-app/model/breeze/entities/generated/entity-model' or its corresponding type declarations.`  
The Angular application compiles just fine so it is only VS Code that does not understand the path and as a result I get no intellisense on the imported classes. 

If that project was not part of an Angular Workspace, it would work just fine. So that problem seem to be related to the fact that there is a root `tsconfig.json` file.

As a workaround, I have modified the root `tsconfig.json` to include the path for each projects. For instance :  

![image](https://user-images.githubusercontent.com/4160415/102065902-6b394b00-3df9-11eb-916e-bf277d22243b.png)

This works if I have the entire workspace opened in VS Code. But since a workspace has so many files, I usually open an instance of VS Code per project, in which case, the root `tsconfig.json` file is no longer considered by VS Code and I loose intellisense again.

So to sum up :  
-  Why VS Code does not consider the `tsconfig.app.json` file specific to each project  ? 
- Why VS Code does not consider the root `tsconfig.json` file if I open the project folder instead of the entire workspace  ? 

Does this issue occur when all extensions are disabled?: Yes

**Comment by mjbvz** [comment]
VS Code only uses project files that are called `tsconfig.json`. You can use the `TypeScript: Go to project configuration` command to investigate which project the currently opened TypeScript file belongs to.

If you still don't think IntelliSense is working  as expected, please share a minimal example project that demonstrates the issue

**Comment by graphicsxp** [comment]
If I select ´TypeScript: Go to project configuration´ when I have the entire Angular workspace opened in VS Code, then the root ´tsconfig.json´ is found;

However, as I suspected,  when I do the same with a single Angular project opened in VS Code, then I get the following popup:

![image](https://user-images.githubusercontent.com/4160415/102186067-5cfb3580-3eb2-11eb-999d-00869fee83fc.png)

I don't know if you understand the problem with a multi-projects Angular workspace (not a VS workspace), but each project has its dedicted ´tsconfig.app.json´ file, that inherits from the root ´tsconfig.json´ file. 

It seems, as you pointed it, that VS Code does not support this and it is only working with files named ´tsconfig.json´ 

But because VS Code is so popular amongst Angular devs, would it be possible to support this scenario  ?

**Comment by mjbvz** [comment]
Tracked by https://github.com/microsoft/TypeScript/issues/33094

Until then, you need to use project files names that are called `tsconfig.json`
